app/backend/src/couchers/phone/sms.py

```diff
@@ -0,0 +1,20 @@
+import secrets
+
+import luhn
+
+from couchers import errors
+
+
+def generate_random_code():
+    """Return a random 6-digit string with correct Luhn checksum"""
+    return luhn.append("%05d" % secrets.randbelow(100000))
```

Comment:
Please move this function into `crypto.py` (or just `"%05d" % secrets.randbelow(100000)` if you want to keep the luhn stuff here).

I want to keep all the cryptography functions in one place so they're easy to verify and audit!